### DISCUSSION TITLE
［委托］🙏🙏🙏

### DISCUSSION THREAD

**Discussion by yansai0v0** [share121/inter-knot#5095] [绝区零]
可以推荐一些好看的番吗

**Comment by dzqwe** [comment]
立番可以吗(´;︵;`)（误   
![这些应该够了](https://s3.bmp.ovh/imgs/2024/09/17/2fafa04cc6afce74.jpg)

**Reply by yansai0v0** [reply]
😨

**Comment by KELERSAMA** [comment]
某科学超电磁炮

**Reply by yansai0v0** [reply]
谢谢，看了

**Comment by Lee9208** [comment]
国家队？

**Comment by keli-star** [comment]
回复术士的重启人生

**Comment by cylca4869** [comment]
轻音少女，辉夜大小姐<img src="https://bbs-static.miyoushe.com/static/2024/09/10/ed624befad4ed330fbdb41c91fc09d10_3535401542533232063.png" alt="飞霄 夸夸" width="75" >

**Comment by 06923** [comment]
缘之空

**Comment by jjxya** [comment]
轻音
孤独
地狱少女
斩！赤红
零之使魔
灼眼的夏娜
魔法少女小圆
夏目
<img src="https://upload-bbs.mihoyo.com/upload/2022/11/14/b171cc5b6c8d8b6257a3bf8163fb2680_567796524796359289.png" alt="阿姬得意" width="75" >

**Reply by jjxya** [reply]
哦对，还有从零开始的异世界生活<img src="https://upload-bbs.mihoyo.com/upload/2022/09/20/80a7ef5a067a1acd495fcc09140b069c_5444302550606416083.gif" alt="爱莉希雅-比心" width="75" >

**Comment by BJ-cmyk** [comment]
死亡笔记！！！💀

**Comment by Fluorine-z** [comment]
海贼王
包打发时间的（bushi

**Comment by BJ-cmyk** [comment]
银魂！！！！

**Comment by YeYingMakas** [comment]
星辉骑士
空洞漫游指南😎

**Comment by TNCNET** [comment]
偶像或者音乐/刑侦类的不知道你是否感兴趣？

前者，比如说LoveLive；偶像大师这俩
后者可以看看ACCA，心理测量者这些

**Comment by Avyere** [comment]
这个委托我接了！
黄段子学生会，野崎君，日常。都挺好看的

**Comment by Sorasakirin** [comment]
心理测量者

**Comment by pwshanjiji** [comment]
摇曳姐妹
![1724563172765](https://github.com/user-attachments/assets/8c823732-12ef-4b91-b5d7-2b9b3689379f)

**Comment by zamsirs** [comment]
eva，全系列

**Comment by hot-ice-seele** [comment]
我心危
美好世界

**Comment by fuqikilroy** [comment]
进击的巨人

**Comment by Loxia1121** [comment]
月刊少女
86
芙莉莲
孤独摇滚

**Comment by qingjiang-neko** [comment]
轻音少女（全系列
孤独摇滚 
mygo
gbc
四大名著全都很好看

**Comment by Doubudou45** [comment]
咸鱼哥！！(〃ﾉωﾉ)

**Comment by wshtdgo** [comment]
我的青春恋爱物语绝对有问题，好看的！

**Reply by wshtdgo** [reply]
而且完结了

**Comment by Kazuha-hhh** [comment]
新疆吐鲁番

**Comment by manyshoes** [comment]
老番的话凉宫，未来日记，罪恶王冠，eva，天降之物
这两年喜欢看乐队番，mygo，少女歌剧，京吹，孤独摇滚，派对浪客
百合轻喜剧比如街角魔族，憧憬成为魔法少女
除了这几个特定类型外，还有86，魔都精兵的奴隶也比较好看，虽然后一个我只追了漫画<img src="https://bbs-static.miyoushe.com/static/2023/05/23/a2fa805a0a782cdb8ab44ddaf8d5343d_962898009752203374.png" alt="阿姬倒地了04" width="75" >

**Comment by area-xk** [comment]
血界战线

**Reply by LiuLiBHZ** [reply]
这个好！

**Comment by RWHwaxili** [comment]
少女与战车

**Comment by HitoriGotoh-Y** [comment]
必须孤独摇滚
